### DISCUSSION TITLE
Welcome to nREPL Discussions!

### DISCUSSION THREAD

**Discussion by bbatsov** [nrepl/nrepl#229] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the nREPL community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by onetom** [comment]
Hi, I'm @onetom!

I'm a regular user of nREPL since 2014, mostly via IntelliJ/Cursive, but I've dabbled a lot with `inf-clojure` and `CIDER` in Emacs too and tried all the available nREPL clients for Atom, Sublime Text, LightTable, vim and VS Code.

I've read most of the docs and skimmed the nREPL source code a few times as of 2025-06-09.

Since Cursive provides code navigation via static source code analysis, I'm not really utilizing a lot of nREPL capabilities, so I don't consider myself a power user of nREPL.

**Reply by bbatsov** [reply]
First introduction here! 🎉
